docs/en/operations/utilities/clickhouse-format.md

```diff
@@ -0,0 +1,99 @@
+---
+toc_priority: 65
+toc_title: clickhouse-format
+---
+
+# clickhouse-format {#clickhouse-format}
+
+The `clickhouse-format` program enables formatting input queries.
+
+Keys:
+
+- `--help` or`-h` — Produce help message
+- `--hilite` — Add syntax highlight with ANSI terminal escape sequences
+- `--oneline` — Format in single line
+- `--quiet` or `-q` — Just check syntax, no output on success
+- `--multiquery` or `-n` — Allow multiple queries in the same file
+- `--obfuscate` — Obfuscate instead of formatting
+- `--seed <string>` — Seed arbitrary string that determines the result of obfuscation
+- `--backslash` — Add a backslash at the end of each line of the formatted query. Can be useful when you copy a query from web or somewhere else with multiple lines, and want to execute it in command line.
```

Comment:
```suggestion
- `--hilite` — Add syntax highlight with ANSI terminal escape sequences.
- `--oneline` — Format in single line.
- `--quiet` or `-q` — Just check syntax, no output on success.
- `--multiquery` or `-n` — Allow multiple queries in the same file.
- `--obfuscate` — Obfuscate instead of formatting.
- `--seed <string>` — Seed arbitrary string that determines the result of obfuscation.
- `--backslash` — Add a backslash at the end of each line of the formatted query. Can be useful when you copy a query from web or somewhere else with multiple lines, and want to execute it in command line.
```